### PR TITLE
Fixed issue

### PR DESCRIPTION
**Pull request by pradeeptosarkar**

### CODE REVIEW

**Comment by pradeeptosarkar** [comment]
fixed 2 issues: one on javascript easy and one on javascrit hard

File: tasks/javascript/easy/sum.js [hunk 1]
```diff
@@ -2,3 +2,11 @@
 
 // TODO: Write a function that returns the sum of two numbers
 
+const num1 = 1;
+const num2 = 2;
+
+// add two numbers
+const sum = num1 + num2;
+
```
Comment (nikohoffren) [inline tasks/javascript/easy/sum.js:10]: Code looks good! However, the issue requested a function that returns a sum of two numbers so you could do that with arrow function:
```bash
const sum = (a, b) => a + b
const num1 = 1
const num2 = 2
console.log('The sum of ' + num1 + ' and ' + num2 + ' is ' + sum(num1, num2) + '.')
```
But it's fine, im still going to merge this. Thanks for contributing!

**Review by nikohoffren — APPROVED** [review]
Code look

**Comment by nikohoffren** [comment]
> fixed 2 issues: one on javascript easy and one on javascript hard

Looks good! You can make a new pull request out of every issue, so it is easier to keep track.

**Comment by nikohoffren** [comment]
## Merged

> This is an automated message from Fork, Commit, Merge [BOT].

Thank you for your contribution! Your pull request has been merged. The files have been reset for the next contributor.

### What's next?

If you're looking for more ways to contribute, I invite you to check out my other projects. Just click [here](https://github.com/nikohoffren?tab=repositories) to find more. These projects contain real issues that you can help resolve. You can also check out the [Influences](https://github.com/nikohoffren/fork-commit-merge#influences) section in the README to find more projects similar to this one.

Also please leave a star to this project if you feel it helped you, i would really appreciate it.

I look forward to seeing your contributions!

**Pull request merged**